core/arch/x86/decode_table.c

```diff
@@ -5044,6 +5920,18 @@ const instr_info_t vex_L_extensions[][3] = {
   },
 };
 
+/****************************************************************************
+* Instructions that differ depending on whether evex-encoded
```

Comment:
style: missing trailing `.` here and in several similar sentences below.

Reply:
Done.